src/mito2/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(let_chains)]
```

Comment:
question: Where do we use this feature?

Reply:
Here: https://github.com/GreptimeTeam/greptimedb/pull/2317/files#diff-ba07daefb832b3da3ad8fc85baafdc7c020a20b0ed992637c97ffb3898338a29R86